client/network/src/behaviour.rs

```diff
@@ -417,7 +417,7 @@ impl<B: BlockT, H: ExHashT> NetworkBehaviourEventProcess<block_requests::Event<B
 				self.events.push_back(BehaviourOut::InboundRequest {
 					peer,
 					protocol: self.block_requests.protocol_name().to_owned().into(),
-					result: Ok(total_handling_time),
+					result: Ok(Some(total_handling_time)),
```

Comment:
The whole `impl` block will be gone with https://github.com/paritytech/substrate/pull/7478/. Given the change to `InboundRequest` this needs to be wrapped in a `Some` for now.